dashboard/test/factories/curriculum_factories.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  display_name      :string(255)
+#  key               :string(255)      not null
+#  course_version_id :integer          not null
+#  content           :text(65535)
+#  position          :integer          not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+FactoryGirl.define do
+  factory :reference_guide do
+    display_name "reference guide"
+    key "category/reference-guide"
+    course_version_id 1
```

Comment:
This seems like it might cause problems. Should we be trying to set up a course_version with a factory for this to be connected to?

Reply:
Here is an example of what is happening for resource which is also associated with course_version https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories/factories.rb#L946-L950